.dockerignore

```diff
@@ -1 +1,2 @@
 /dist
+/testdata/gotmp
```

Comment:
For documentation purposes: this stores in the current directory the GOPATH and the GOCACHE. This way `go` can reuse the same files in subsequent invocations via Docker. Of course, this makes Docker invocations faster.